src/aind_metadata_service/tars/mapping.py

```diff
@@ -223,7 +223,7 @@ def get_virus_strains(response: ModelResponse) -> List:
                         and procedure.injection_materials
                     ):
                         virus_strains = [
-                            getattr(material, "name")
+                            getattr(material, "name").strip(" \t\n\r")
```

Comment:
`strip()` will strip white spaces on either side.

Reply:
we also need to strip "\t" and potentially other special characters. This strip handles both leading and trailing spaces though

Reply:
According to Google, the `strip()` method removes leading and trailing whitespace characters, including newlines, spaces, and tabs.